packages/styles/src/theming/sap_horizon.scss

```diff
@@ -95,7 +95,7 @@
   /* Popover */
   --fdPopover_Border_Radius: var(--sapPopover_BorderCornerRadius);
   --fdPopover_Border_Width: 0.0625rem;
-  --fdPopover_Border_Color: #acb8c1;
+  --fdPopover_Border_Color: var(--sapNeutralBorderColor);
```

Comment:
this should be done also for horizon_dark; horizon_hcb and horizon_hcw.
for all SAP components this variable looks better, but looks worse for Vertical Nav and User Menu.
@mikerodonnell89  can you check with Paige what variable to use for those 2 BTP components?